pollination/honeybee_radiance/schedule.py

```diff
@@ -0,0 +1,25 @@
+from dataclasses import dataclass
+from pollination_dsl.function import Function, command, Inputs, Outputs
+
+
+@dataclass
+class EPWtoDaylightHours(Function):
+    """Convert EPW to EN 17037 schedule as a CSV file.
+    
+    This function generates a valid schedule for EN 17037, also known as daylight hours.
+    Rather than a typical occupancy schedule, the daylight hours is half the year with
+    the largest quantity of daylight.
+    """
+
+    epw = Inputs.file(
+        description='Path to epw file.', path='weather.epw'
```

Comment:
It might be good to add a validation here for file extension. Especially to make sure people won't provide a wea file.

Reply:
Good idea. I added the validation!